llvm/lib/ExecutionEngine/Orc/TargetProcess/JITLoaderVTune.cpp

```diff
@@ -13,6 +13,7 @@
 #include "llvm/ExecutionEngine/Orc/Shared/VTuneSharedStructs.h"
 
 #if LLVM_USE_INTEL_JITEVENTS
+#include <map>
 #include "IntelJITEventsWrapper.h"
 #include "ittnotify.h"
```

Comment:
the C++ linter wants you to order this like this though :shrug
```suggestion
#include "IntelJITEventsWrapper.h"
#include "ittnotify.h"
#include <map>
```